apps/src/gamelab/GameLab.js

```diff
@@ -48,9 +48,16 @@ import {
 } from '../containedLevels';
 import { hasValidContainedLevelResult } from '../code-studio/levels/codeStudioLevels';
 import {actions as jsDebugger} from '../lib/tools/jsdebugger/redux';
+import project from '@cdo/apps/code-studio/initApp/project';
 
 var MAX_INTERPRETER_STEPS_PER_TICK = 500000;
 
+// The width and height in pixels of the thumbnail image to capture.
+const THUMBNAIL_SIZE = 180;
+
+// Number of ticks after which to capture a thumbnail image of the play space.
+const CAPTURE_TICK_COUNT = 250;
```

Comment:
I've measured this on my 2012 macbook, and 250 ticks appears to be between 1.0 and 1.25 seconds for an app which draws a single sprite every time `draw()` is called. The ticks in question are interpreter ticks, which may not correspond directly to calls of the gamelab draw loop.